.github/actions/create_manifest/action.yml

```diff
@@ -35,7 +35,7 @@ runs:
       id: create_manifest
       shell: ${{ runner.os == 'Windows' && 'pwsh' || 'bash' }}
       run: >-
-        python ${{ env.ACTION_PATH }}/create_manifest.py
+        python3 ${{ env.ACTION_PATH }}/create_manifest.py
```

Comment:
Ubuntu 20 failed with `python: command not found` as I've removed the virtual env from its Docker and used the system Python there. I guess we can explicitly use `python3` as we already do in all steps across workflows.